packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -778,13 +778,15 @@ function isVirtualPointerEvent(event: PointerEvent) {
   // Android TalkBack double tap will sometimes return a event with width and height of 1
   // and pointerType === 'mouse' so we need to check for a specific combination of event attributes.
   // Cannot use "event.pressure === 0" as the sole check due to Safari pointer events always returning pressure === 0
-  // instead of .5, see https://bugs.webkit.org/show_bug.cgi?id=206216
+  // instead of .5, see https://bugs.webkit.org/show_bug.cgi?id=206216. event.pointerType === 'mouse' is to distingush
+  // Talkback double tap from Windows Firefox touch screen press
   return (
     (event.width === 0 && event.height === 0) ||
     (event.width === 1 &&
       event.height === 1 &&
       event.pressure === 0 &&
-      event.detail === 0
+      event.detail === 0 &&
+      event.pointerType === 'mouse'
```

Comment:
wonder if there is a problem with stylus pens (pointerType === 'pen')
does anyone on the team have a stylus? i do not

Reply:
I know of nowhere that checks pen, https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pointerType

Reply:
I don't think it should matter because this is to detect TalkBack which always fires events with pointerType=mouse